### DISCUSSION TITLE
Fixing an exsisting icon

### DISCUSSION THREAD

**Discussion by digitaldirk** [Arcticons-Team/Arcticons#1839] [Q&A] [answered]
Hi,

1. I need to update the app's existing svg file name, I only need to do that in icons/black and icons/white? I saw other places where the svg's name was but I assumed those are generated?
2. In this case the app has the same activity but a different drawable name, in my PR will I keep the old activity entry for people who may still have that version or get rid of it due to that version not being available anymore?
3. Anything else I am missing? (apart from updating the svg names, and the appfilter file)

Thank you

**Comment by Donnnno** [comment, marked as answer]
Right now, there are 5 files that you have to rename:
![image](https://github.com/Donnnno/Arcticons/assets/31142286/614f5a6d-3468-4481-ab92-7ece5fbf7b5a)
And you'll have to change the drawable name in the appfilter.xml in /other

**Reply by digitaldirk** [reply]
Awesome thank you!

**Reply by digitaldirk** [reply]
For future reference: https://github.com/Donnnno/Arcticons/issues/1840
